lgc/state/Compiler.cpp

```diff
@@ -178,25 +173,14 @@ void PipelineState::generate(std::unique_ptr<Module> pipelineModule, raw_pwrite_
     replayerPass = createBuilderReplayer(this);
 
   // Patching.
-  Patch::addPasses(this, *patchPassMgr, replayerPass, patchTimer, optTimer, checkShaderCacheFunc);
+  Patch::addPasses(this, *passMgr, replayerPass, patchTimer, optTimer, checkShaderCacheFunc);
 
   // Add pass to clear pipeline state from IR
-  patchPassMgr->add(createPipelineStateClearer());
-
-  // Run the "whole pipeline" passes, excluding the target backend. Stop here if the pass manager was stopped
-  // by -emit-lgc
-  patchPassMgr->run(*pipelineModule);
-  if (m_emitLgc)
```

Comment:
Does -emit-lgc still work? I think it should; the idea is that the "passMgr->stop()" above stopped any further passes being added.

Reply:
I hope it still works but I don't know how to test it. I suppose I can try. Is there anything in the automated test suite that would fail if I broke it?

Reply:
No automated testing, but "amdllpc -emit-lgc -o -" then use the stdout as input to "lgc".